src/System.Device.Gpio/System/Device/Pwm/Channels/Windows10PwmChannel.Windows.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.Devices.Enumeration;
+using Windows.Security.ExchangeActiveSyncProvisioning;
+using WinPwm = Windows.Devices.Pwm;
+
+namespace System.Device.Pwm.Channels
+{
+    /// <summary>
+    /// Represents a PWM channel running on Windows 10 IoT.
+    /// </summary>
+    internal partial class Windows10PwmChannel : PwmChannel
+    {
+        private WinPwm.PwmController _winController;
+        private WinPwm.PwmPin _winPin;
+        private int _frequency;
+        private double _dutyCyclePercentage;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Windows10PwmChannel"/> class.
+        /// </summary>
+        /// <param name="chip">The PWM chip number.</param>
+        /// <param name="channel">The PWM channel number.</param>
+        /// <param name="frequency">The frequency in hertz.</param>
+        /// <param name="dutyCyclePercentage">The duty cycle percentage represented as a value between 0.0 and 1.0.</param>
+        public Windows10PwmChannel(
+            int chip,
+            int channel,
+            int frequency = 400,
+            double dutyCyclePercentage = 0.5)
+        {
+            // When running on Hummingboard we require to use the default chip.
+            var deviceInfo = new EasClientDeviceInformation();
```

Comment:
`EasClientDeviceInformation` this looks like it has typo in the name

Reply:
Nope, this is a WinRT class. I will adress your feedback in another change I’m about to push which also contains updates to Readme.md files and xml comments for the bindings we changed.

Reply:
I've already pushed xml change too since it was breaking the build. add whatever else is needed (if possible try to make it so that we have only 3 commits in the end but honestly we can as well just merge this branch as is)